dz/login.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+	<meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <title>Login</title>
+    <link rel="stylesheet" type="text/css" media="screen" href="css/common.css"/>
+    <link rel="stylesheet" type="text/css" media="screen" href="css/loginStyle.css"/>
+</head>
+<body>
+    <div class="container center with-shadow">
+		<form class="main-block">
+            <h2 class="title">Логин</h2>
+            <div class="button-social">
+                <button class="btn vkontakteButton flex-center"><img src="images/vk.svg" />Войти через Вконтакте</button>

```

Comment:
Давай для читаемости разнесём эту строку на несколько:
```suggestion
                <button class="btn vkontakteButton flex-center">
                    <img src="images/vk.svg" />
                    Войти через Вконтакте
                </button>
```

Так будет глазами в разметке проще отыскать текст "Войти через Вконтакте", если кому-то понадобится найти эту кнопку.